djconnectwise/sync.py

```diff
@@ -296,6 +296,13 @@ def update_or_create_instance(self, api_instance):
                   " Error: {}".format(self.model_class, e)
             logger.error(msg)
             raise InvalidObjectException(msg)
+        except ValueError as e:
+            # ContactCommunication syncs don't fail with a record
+            # that has this kind of invalid data
+            msg = "ValueError while attempting to sync {}." \
+                  " Error: {}".format(self.model_class, e)
+            logger.error(msg)
```

Comment:
This is not what I meant. We want to change the `ContactCommunication` model so it doesn't accept null or blank values at all. You need to change the actual model record:

From:
```
class ContactCommunication(models.Model):
value = models.CharField(blank=True, null=True, max_length=250)
```

To:
```
class ContactCommunication(models.Model):
value = models.CharField(max_length=250)
```

Reply:
That way, if a record is saved with invalid data it is automatically caught with the current process.

Reply:
Oh, I need coffee this time. Thanks Sam.

Reply:
You can review this again